journal/modernized/32-epistles.adoc

```diff
@@ -115,10 +120,12 @@ and in the end produce both peace and comfort.
 My dear love is to you, as a true and faithful husband, which you may assure yourself of,
 wherever I am, or however I may be disposed of;
 and my tender love is to all honest-hearted Friends, as if I named them, and rest
+//after "rest" above, there's no punctuation in the PDF, is that okay?
 
-Your loving husband.
+[.signed-section-signature]
+Your loving husband
 
-=== Part of a Letter written by William Edmundson, at Barbadoes, to Friends in Ireland; dated the 8th of the First month, 1675.
+=== Part of a Letter written by William Edmundson, at Barbados, to Friends in Ireland; dated the 8th of the First month, 1675.
```

Comment:
Ok, here is a place where it is pretty confusing. This is technically a [.letter-heading], because it describes the contents and/or recipients of a letter. BUT... since letter headings are rendered in bold and in all caps, it starts to look kinda goofy/ugly to have LONG BLURBY letter headings in **bold** and ALL CAPS.  So in this case (and in the other one that Jared noted) I think it would just look cleaner to label this with a [.blurb] tag instead of the [.letter-heading] tag.  Does that make sense??  Like this:

```ASCIIDOC
[.blurb]
=== Part of a Letter written by William Edmundson, at Barbados, to Friends in Ireland; dated the 8th of the First month, 1675.
```
But this is only because of how it LOOKS when it is rendered in a book. It's really a letter heading, but it looks better to mark the really long letter headings as blurbs.  (so confusing!!)